snip-gui/src/app/Components/navigation-toolbar/navigation-toolbar.component.html

```diff
@@ -0,0 +1,19 @@
+<mat-toolbar>
+  <mat-card>
```

Comment:
Can you take out the mat-card and use spans? Its should help it render as a proper header. [Examples here:](https://material.angular.io/components/toolbar/overview)

Reply:
I was wondering why that input field was so hard to style :) updating now, thanks for showing me that